src/shogun/preprocessor/RandomFourierGaussPreproc.h

```diff
@@ -22,44 +22,21 @@ namespace shogun {
  *
  * approximation quality depends on dimension of feature space, NOT on number of data.
  *
- * effectively it requires two parameters for initialization: (A) the dimension of the input features stored in
- * dim_input_space
+ * effectively it requires only two parameters for initialization: 
+ * (A) the width of the kernel
  * (B) the dimension of the output feature space
- *
- * in order to make it work there are two ways:
- * (1) if you have already previously computed random fourier features which you want to use together with
- * newly computed ones, then you have to take the random coefficients from the previous computation and provide them
- * via void set_randomcoefficients(...) for the new computation
- * this case is important for example if you compute separately features on training and testing data in two feature objects
- *
- * in this case you have to set
- * 1a) void set_randomcoefficients(...)
- *
- * (2) if you compute random fourier features from scratch
- * in this case you have to set
- * 2a) set_kernelwidth(...)
- * 2b) void set_dim_feature_space(const int32_t dim);
- * 2c) set_dim_input_space(const int32_t dim);
- * 2d) init_randomcoefficients() or apply_to_feature_matrix(...)
  */
 class RandomFourierGaussPreproc: public RandomMixin<DensePreprocessor<float64_t>> {
 public:
 	/** default constructor */
 	RandomFourierGaussPreproc();
 
-	/** alternative constructor */
-	RandomFourierGaussPreproc(const RandomFourierGaussPreproc & pr);
-
-	/** default destructor
-	 * takes care for float64_t* randomcoeff_additive,float64_t* randomcoeff_multiplicative;
-	 */
+	/** default destructor */
```

Comment:
remove

Reply:
(the comment)

Reply:
it's actually not resolved. Please don't mark things as resolved if they are not! Thanks :)

Reply:
I have made the changes in my local repo, I resolve them beforehand here so that to improve my focus. After I push all this will be updated at one go.